### ISSUE TITLE
Accuracy benchmarks and official QA process?

### ISSUE THREAD

**Issue by benhalpern** [jamespilgrim/PiTrac#17]
I don't think this is an immediate concern, but once there is some momentum in the community I think it could make sense to have an official process and recommendation for verifying improvements and regressions?

I'm thinking something along the lines

1. Automated testing against a static stash of images with known outcomes
2. Manual QA testing checklist instructions for IRL swing testing accompanied by non-interfering third-party LMs
3. Automated performance benchmarks

I imagine this could create a powerful feedback loop for community QA contribution if the process is laid out in an easy-to-follow formula.

**Comment by jamespilgrim** [comment]
Agreed - and there's a start of the code to do this, but so much has
changed lately that I need to rebuild all the tests with fresh images and
expected results data.

**Comment by jeshernandez** [comment]
I can help setting up some jobs, unit test validation, pre-build, etc. I just need to know if we're using `[ubuntu-latest]` as the runner? Also, permissions to run jobs (read only)? I pay around $4 (per seat) for a team license and able to use 3,000 minutes of Action runners.
